server/issue.go

```diff
@@ -518,24 +521,25 @@ func (p *Plugin) httpGetJiraProjectMetadata(w http.ResponseWriter, r *http.Reque
 		}
 	}
 
-	projects := []utils.ReactSelectOption{}
-	issues := map[string][]utils.ReactSelectOption{}
-	for _, prj := range plist {
-		projects = append(projects, utils.ReactSelectOption{
-			Value: prj.Key,
-			Label: prj.Name,
-		})
-		issueTypes := []utils.ReactSelectOption{}
-		for _, issue := range prj.IssueTypes {
-			if issue.Subtask {
+	type option = utils.ReactSelectOption
+	projects := make([]option, len(metaInfo.Projects))
+	issues := make(map[string][]option, len(metaInfo.Projects))
+	for index, project := range metaInfo.Projects {
+		projects[index] = option{
+			Value: project.Key,
+			Label: project.Name,
+		}
+		issueTypes := make([]option, len(project.IssueTypes))
+		for index, issueType := range project.IssueTypes {
+			if issueType.Subtasks {
 				continue
 			}
-			issueTypes = append(issueTypes, utils.ReactSelectOption{
-				Value: issue.ID,
-				Label: issue.Name,
-			})
+			issueTypes[index] = option{
```

Comment:
Are these changes relevant to the purpose of this PR? Also it seems we may introduce a bug here, since the `continue` statement above will result in a null pointer on the frontend, since we are "skipping" over that index.